### ISSUE TITLE
neurodata microns branch compatibility

### ISSUE THREAD

**Issue by wrgr** [openconnectome/ndio#100]
when server exists, write microns.py remote.

ocp -> nd

**Comment by j6k4m8** [comment]
what timeline is this..? wondering if we should close and re-open closer to a launch

**Comment by wrgr** [comment]
I just tagged - let's leave open, but no action for now, and only relevant once ndio v.1.0 is ready.

**Comment by Aeusman** [comment]
Closing in favor of individual issues for broken APIs from microns switch since most are not broken.
